devito/passes/iet/languages/openacc.py

```diff
@@ -183,7 +183,10 @@ def _make_partree(self, candidates, nthreads=None):
             # TODO: still unable to exploit multiple par-tiles (one per nest)
             # This will require unconditionally applying blocking, and then infer
             # the tile clause shape from the BlockDimensions' step
-            tile = self.par_tile[0]
+            par_tile_length = len(self.par_tile)
+            idx = index if index < par_tile_length else par_tile_length - 1
```

Comment:
we prefer explicit if-then-else:

```
if ...:
    idx = ...
else:
    idx = ...
```

Reply:
and in practice this is just 

```
index = min(index, par_tile_length-1)
```

Reply:
Which one is more suitable?
```
tile = self.par_tile[min(index, len(self.par_tile) - 1)]
```

```
idx = min(index, len(self.par_tile) - 1)
tile = self.par_tile[idx]
```

Reply:
I'd say the latter for homogeneity with the rest of the codebase.